### DISCUSSION TITLE
Tag position

### DISCUSSION THREAD

**Discussion by Ekaterinador** [larics/icuas22_competition#64] [Q&A]
Good afternoon! I would like to ask if there is a maximum and minimum height for the tag (perhaps the limit is less than the height of the walls)? And a similar question for X and Y position of the tag.

**Comment by fpetric** [comment]
Tags can be placed only in Zone3, so x and y limits can be extracted from zone boundaries. Heightwise, tag can be anywhere on the wall (0, 4.5m), but in practice we will not put them very close to the floor or the "ceiling".

**Reply by Ekaterinador** [reply]
Thank you!
